torch/csrc/jit/codegen/cuda/expr_evaluator.cpp

```diff
@@ -53,7 +53,7 @@ void EvaluationContext::print() const {
       std::cout << " ; original value = "
                 << kv.first->as<Int>()->value().value();
     }
-    std::cout << "\n";
+    std::cout << " ; " << *kv.first->getValType() << "\n";
```

Comment:
why do we need this?

Reply:
I have to bind values for the Fusion IR and Kernel IR. Printing the type allows me to distinguish between them.